lnwallet/wallet.go

```diff
@@ -543,6 +550,50 @@ func (l *LightningWallet) PsbtFundingVerify(pid [32]byte,
 		return fmt.Errorf("error verifying PSBT: %v", err)
 	}
 
+	// Get the channel reservation for that corresponds to this pending
+	// channel ID.
+	l.limboMtx.Lock()
+	pid, ok := l.reservationIDs[pendingChanID]
+	if !ok {
+		l.limboMtx.Unlock()
+		return fmt.Errorf("no channel reservation found for "+
+			"pendingChannelID(%x)", pendingChanID[:])
+	}
+
+	pendingReservation, ok := l.fundingLimbo[pid]
+	l.limboMtx.Unlock()
+
+	if !ok {
+		return fmt.Errorf("no channel reservation found for "+
+			"reservation ID %v", pid)
+	}
+
+	// Now the the PSBT has been verified, we can again check whether the
+	// value reserved for anchor fee bumping is respected.
+	numAnchors, err := l.currentNumAnchorChans()
+	if err != nil {
+		return err
+	}
+
+	// If this commit type is an anchor channel we add that to our counter,
+	// but only if we are contributing funds to the channel. This is done
+	// to still allow incoming channels even though we have no UTXOs
+	// available, as in bootstrapping phases.
+	if pendingReservation.partialState.ChanType.HasAnchors() &&
```

Comment:
Yes, but  feel it could be very disruptive to disallow incoming channels for new nodes, at least for now. We could make this stricter later on.

Reply:
@Roasbeef made a good point offline, it's not as big of deal if the node isn't routing because you don't have to take action before the incoming timelock. one option here is to allow incoming channels if `reject_htlc=1`